main.go

```diff
@@ -29,5 +30,12 @@ func main() {
 	prometheus.MustRegister(opsgenieCollector)
 
 	http.Handle("/metrics", promhttp.Handler())
-	log.Fatal(http.ListenAndServe(":8000", nil))
```

Comment:
`
G114: Use of net/http serve function that has no support for setting timeouts
`